### ISSUE TITLE
Coral Dev Mini did not bootup properly anymore

### ISSUE THREAD

**Issue by Crijstl4n167** [google-coral/edgetpu#727]
### Description

Hello,
for many weeks we used Coral Dev Mini and VisualStudioCode with remote-ssh for remote development.
Some days ago our problems started.
It was possible to connect via ssh but VisualStudioCode could not start/debug anymore.
And now the Dev Mini does not start properly - ssh connect is not possible.
We conneted SerialConsole, we see messages and errors but cannot login:

```
[10:02:22:744] [    0.189582] PVR_K:(Error):     1: mtk_mfg_async_init: Enable power domain failed␍␊
[10:02:22:744] [    0.189786] PVR_K:(Error):     1: mtk_mfg_2d_init: Enable power domain failed␍␊
...
[10:29:59:105] [    3.713412] systemd[1]: /lib/systemd/system/vitalsd.service:3: Failed to add dependency on usb-gadget, ignoring: Unknown error -22...
[10:30:00:179] [    4.787545] EXT4-fs (mmcblk0p4): warning: mounting fs with errors, running e2fsck is recommended␍␊
[10:30:00:190] [    4.796974] EXT4-fs (mmcblk0p4): re-mounted. Opts: (null)␍␊

```

Tried to recover with "fastboot":
```
christian@w520L:~/Downloads/excelsior-eagle-20201210233645$ bash enable_lk_fastboot.sh 
**********************************************************************
Unable to connect to debug board, check connection or manually toggle GPIO.
**********************************************************************
/tmp/tmp.BGhe2jCqlU ~/Downloads/excelsior-eagle-20201210233645
INFO: pySerial version: (3.5)
INFO: Use config file: /tmp/tmp.BGhe2jCqlU/dl_addr.ini
INFO: Waiting to connect platform...
INFO: Got /dev/ttyACM3
INFO: Connect brom
INFO: Loading file: lk.bin
INFO: Send lk.bin
INFO: Jump da
~/Downloads/excelsior-eagle-20201210233645

```
No change. 
During "fastboot" strange logs:
```
[10:27:37:128] ␍                                                   ␍[DL] 00001388 00000000 010303␊
[10:28:13:770] ␍<0x18><0x84>i<0x18><0xf8>9<0x8c><break>
[10:28:14:813] <0x01><0xb5><0xa2><0x14><0x80><0x90>i<0x80><0x90><0x04><break>
[10:28:14:813] <0x84>%<0x04>⇥	<0xb9><0x02><0x80><break>
[10:28:14:813] <0x88><0x1c><0xc0>a<0xa9><0x84>#⇥	<0x84><0x84><0x9c>#⇥	<0x84><0x11><0xc4><0x0c>0<0x1c><break>
...
```

Does anybody have an idea how to get the board working again?
Thank you!

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Build/Install

### Operating System

Mendel Linux

### Coral Device

Dev Board Mini

### Relevant Log Output

_No response_</details>

**Comment by Crijstl4n167** [comment]
Here are my complete serial console outputs:

Normal bootup output:
[02_normal_bootup.txt](https://github.com/google-coral/edgetpu/files/10830756/02_normal_bootup.txt)

Output while holding testpin to ground:
[01_enable_fastboot_log.txt](https://github.com/google-coral/edgetpu/files/10830758/01_enable_fastboot_log.txt)

**Comment by Crijstl4n167** [comment]
Solved.
Did fastboot. Used the wrong USB-C for connecting the PC :-o
So the fastboot script did not see /dev/ttyACM3.
Using the correct USB-C connector and everything is exactly as written in the original [Coral.ai docs](https://coral.ai/docs/dev-board-mini/reflash/#force-boot-into-fastboot-mode).
I used the "bash flash.sh" script without the switch "-H".
Then it took some time and now the board is online again :-)
